layer1/transaction/backend.go

```diff
@@ -89,13 +89,13 @@ func newReplacedInfo(newTxn *types.Transaction, originalTxInfo info) info {
 	)
 }
 
-// Internal struct to keep track of transactions retries groups
+// group is an internal struct to keep track of transactions retries groups
```

Comment:
Internal struct is implied.

```suggestion
// group to keep track of transactions retries.
```